### ISSUE TITLE
Percussion panel 'Write/Preview' buttons do not state 'Selected'

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32783]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

In the Percussion Panel the "write" and "preview" buttons do not announce their state as selected.  This means a blind user has no way of knowing when they press enter on various pads whether they are hearing a preview or writing to the score.

STEPS:
1.  Create a score with a drum kit and navigate to that staff.
2. Press Tab 2 or 3 times to get to the percussion panel and once more to hear "write button".
3. Press an arrow key to hear "Preview button".
Notice that neither button announced its state as selected.
4.  Press enter to select the Preview button and arrow off and on it and note still no feedback to show it has been selected.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jmwmusic** [comment]
I have just retested this in 4.7 and there seems to be no change.  The buttons still do not announce their state to the screen reader.

**Comment by MarcSabatella** [comment]
It seems the PR that fixes this was merged to master (which will become 5.0) but not 4.7.  I don't know how viable it is to include it in a 4.7.x patch, but it seems worth considering.
